examples/simple_closure.rs

```diff
@@ -0,0 +1,57 @@
+//! Simple Goose load test example using closures.
+//!
+//! ## License
+//!
+//! Copyright 2020 Fabian Franz
+//!
+//! Licensed under the Apache License, Version 2.0 (the "License");
+//! you may not use this file except in compliance with the License.
+//! You may obtain a copy of the License at
+//!
+//! http://www.apache.org/licenses/LICENSE-2.0
+//!
+//! Unless required by applicable law or agreed to in writing, software
+//! distributed under the License is distributed on an "AS IS" BASIS,
+//! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//! See the License for the specific language governing permissions and
+//! limitations under the License.
+
+use goose::prelude::*;
+
+use std::boxed::Box;
+use std::sync::Arc;
+
+fn main() -> Result<(), GooseError> {
+    let mut taskset = taskset!("WebsiteUser")
+        // After each task runs, sleep randomly from 5 to 15 seconds.
+        .set_wait_time(5, 15)?;
+
+    let paths = vec!["/", "/about", "/our-team"];
+    for request_path in paths {
+        // Clone the string so that it can be moved into the closure.
+        let path = request_path;
```

Comment:
Are we now cloning the reference, not the string?

Reply:
We actually don't need to clone as I used paths and not &paths now ...